packages/types/src/user.ts

```diff
@@ -66,6 +65,8 @@ export type Self = {
   updatedAt: string;
 };
 
+export type PopulatedSelf = Self & { online: boolean };
```

Comment:
I don't think we need this type. Let's use just `IUser.Self`. The frontend doesn't use the `online` field (if it does, you can disable/remove it). The frontend will access the online filed via the event.

Reply:
Shall I remove `PopulatedSelf` completely, or just leaving it equals `Self` (removing the union) for future modifications?